index.html

```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
           </p>
         </div>
         <div class="project-tag">
+          <ul class="project-skills flex">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+        <button class="see-project-btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>
+  <section id="about" class="about">
```

Comment:
There is more space between the title of the section and the top of the section. You can try to approach to a total of `114px` using flex sizing and margin etc.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
|  ![image](https://user-images.githubusercontent.com/61361037/219856679-d4de179a-63ec-4360-a6d6-82f84ec107d5.png)| ![image](https://user-images.githubusercontent.com/61361037/219856748-1ed06384-4635-48e7-a298-9bf755d94e6c.png)|